config/locales/en.yml

```diff
@@ -96,11 +124,10 @@ en:
     bibliographic_id: |
       Bibliographic ID should be used for an external record identifier that can connect the Avalon item to a catalog record or other record for the original item. Type specifies the type of external record identifier.
     language: |
-      Language describes the languages used in the item. Typing two or more
+      <strong>Required field.</strong>Language describes the languages used in the item. Typing two or more
       letters within the field will bring up permitted matches. Only terms or
       codes from the <a href="http://www.loc.gov/marc/languages/"
-      target="_blank">MARC Code List for Languages</a> may be used. This is an
-      optional field.
+      target="_blank">MARC Code List for Languages</a> may be used. Hint: for English language materials, type "English".
```

Comment:
Any <a> tag with `target="_blank"` needs a `rel="noopener noreferrer"` for security purposes:
https://dev.to/ben/the-targetblank-vulnerability-by-example

(arguably the Library of Congress isn't very untrustworthy, but then again, it has _Congress_ in its name)

Reply:
Done for all of these in the locales file.